contents/handbook/why-does-posthog-exist.md

```diff
@@ -6,29 +6,63 @@ showTitle: true
 
 ## Our mission
 
-Equip every developer to build successful products
+Equip every developer to build successful products.
+
+## A few words
+
+As you know, the initial products of PostHog are a bunch of different SaaS tools, which work together better than everyone else’s. However, what most people may not know, is that our long term plan is to automate how entire businesses operate built around the unit they operate on - their customer data.
```

Comment:
This is the very first chapter, so "As you know" makes me wonder if I've missed something.

```suggestion
On the surface, PostHog's products are a bunch of different SaaS tools, which work together better than anyone else’s. However, the secret to our success is our long term plan: automate how entire businesses operate by building around the unit they operate on - their customer data.
```

I don't like the "secret to our success" language but something along those lines..

Reply:
agree - think the secret thing is slightly the wrong ligno, i'll incorporate this then remove that